platforms/js/build_js.py

```diff
@@ -194,6 +198,7 @@ def build_doc(self):
     parser.add_argument('--skip_config', action="store_true", help="Skip cmake config")
     parser.add_argument('--config_only', action="store_true", help="Only do cmake config")
     parser.add_argument('--enable_exception', action="store_true", help="Enable exception handling")
+    parser.add_argument('--custom_flags',help="Change modules settings") 
```

Comment:
Please add a single whitespace after a comma.

Reply:
I think we need to have more detailed description. These are extra CMake flags so it'd be better to describe it.